h/views/account_signup.py

```diff
@@ -62,6 +62,7 @@ def post(self):
         except ConflictError as exc:
             template_context["heading"] = _("Account already registered")
             template_context["message"] = _(
+                # pylint:disable=consider-using-f-string
                 "{failure_reason}".format(failure_reason=exc.args[0])
             )
```

Comment:
```suggestion
            template_context["message"] = _(str(exc.args[0]))
```
?

Reply:
I think the right fix here is removing the `_()` (what's the translation of a dyncamic `exc.arg[0]`?) but not 100% sure so I left it alone.